packages/flutter_tools/lib/src/ios/simulators.dart

```diff
@@ -187,7 +212,7 @@ class SimControl {
 
   Future<void> takeScreenshot(String deviceId, String outputPath) async {
     try {
-      await processUtils.run(
+      await _processUtils.run(
         <String>[_xcrunPath, 'simctl', 'io', deviceId, 'screenshot', outputPath],
         throwOnError: true,
       );
```

Comment:
I'm not sure failing to take a screenshot should throw a tool exit. Maybe it should just print an error and continue?

Reply:
Done.